src/idd.ml

```diff
@@ -16,15 +17,16 @@ type manager = {
   seq_cache : (Pair.t, t) Hashtbl.t;
 }
 
-let manager () : manager = {
-  dd = Dd.manager ();
+
+let manager ?d_mgr:(d_mgr=Dd.manager ()) () : manager = {
+  dd = d_mgr;
   union_cache = Hashtbl.create (module Pair);
   seq_cache = Hashtbl.create (module Pair);
 }
 
-let test mgr i b =
```

Comment:
See above. I think I'd rather have duplicate code in bdd.ml/idd.ml then put this function into dd.ml. It's just 2 lines. Wdyt?